src/main/java/com/googlecode/jsonrpc4j/spring/AbstractJsonServiceExporter.java

```diff
@@ -38,8 +38,9 @@
 	 * {@inheritDoc}
 	 */
 	public void afterPropertiesSet() throws Exception {
+		objectMapper = null;
 
-		if (objectMapper == null && applicationContext != null && applicationContext.containsBean("objectMapper")) {
+		if (applicationContext != null && applicationContext.containsBean("objectMapper")) {
```

Comment:
What's the purpose of removing `objectMapper == null` from this if statement and instead setting it to `null` on line 41? What is the value of `objectMapper` when `afterPropertiesSet()` is called?